Modules/Survey/classes/class.ilSurveySkill.php

```diff
@@ -247,6 +247,8 @@ function determineSkillLevelsForAppraisee($a_appraisee_id, $a_self_eval = false)
 	{
 		$skills = array();
 
+		$log = ilLoggerFactory::getRootLogger("svy");
```

Comment:
getRootLogger does not have any parameters. I guess you mean getLogger

Why not put this in a member var in the constructor like in other cases?
$this->log = ilLoggerFactory::getLogger("svy");